sway/main.c

```diff
@@ -373,7 +373,7 @@ int main(int argc, char **argv) {
 	// prevent ipc from crashing sway
 	signal(SIGPIPE, SIG_IGN);
 
-	sway_log(SWAY_INFO, "Starting sway version " SWAY_VERSION);
+	sway_log(SWAY_INFO, "Starting swayfx version " SWAY_VERSION " (based on sway version " SWAY_ORIGINAL_VERSION ")");
```

Comment:
Reduce line length

Reply:
not entirely sure about the formatting style for this one, i can rebase the last commit if it looks alright